### DISCUSSION TITLE
s/2023-devops-service-platform-tools

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#40] [Announcements]
# s/2023-devops-service-platform-tools

这一年我们上线的运维自动化系统

https://blog.ops-coffee.cn/s/2023-devops-service-platform-tools

**Comment by 7gugu** [comment]
很厉害啊，一年时间就做出这么多的基础建设

**Comment by shimingsir** [comment]
这个效果展示让我很震撼

**Comment by JvvYou** [comment]
有些功能是不是跟云平台上重复了

**Reply by ops-coffee** [reply]
是的，多云基本都是重复的，不过，不一样的是，会隐藏不同云厂商不同的地方，用户也无需了解多云太多的信息，就跟使用一朵云一样用多云，操作会更简单，使用会更方便

**Comment by yuea1755** [comment]
大佬，能求一份域名和硬件资产管理、软件使用的么？我也想学习一下，感觉这些东西真的挺实用的

**Comment by inke100** [comment]
想知道，这些运维系统都是开源么
